test/unit-tests.js

```diff
@@ -131,6 +132,10 @@ describe('grunt-jsonlint task', function () {
     testReformattingFile(3);
   });
 
+  it('reformats the input JSON file with object keys sorted', function () {
+    testSortingObjectKeys();
+  });
+
```

Comment:
It would be nice to have a negative test to verify we only reformat files when instructed to do so.

However, it's not a blocker because the feature is demonstrated to work in such a way that if it broke, the test would surely fail.